src/assets/allowance-module/v0.1.1/allowance-module.json

```
@@ -6,7 +6,8 @@
     "100": "0xAA46724893dedD72658219405185Fb0Fc91e091C",
     "130": "0xAA46724893dedD72658219405185Fb0Fc91e091C",
     "137": "0xAA46724893dedD72658219405185Fb0Fc91e091C",
-    "80094": "0xAA46724893dedD72658219405185Fb0Fc91e091C"
+    "80094": "0xAA46724893dedD72658219405185Fb0Fc91e091C",
+    "232": "0x659336c47dA09889AAC1776Ad45262CD5F6B80C9"
```

nit:

```suggestion
    "232": "0x659336c47dA09889AAC1776Ad45262CD5F6B80C9",
    "80094": "0xAA46724893dedD72658219405185Fb0Fc91e091C"
```

We should leave things in sorting order.